spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -32,6 +32,13 @@
       expect(response.content_type).to eq('application/json')
       expect(response).to match_response_schema('institutions')
     end
+
+    it 'search returns case-insensitive results' do
+      get :index, name: 'CHICAGO', version: 1
```

Comment:
Should we maybe improve this to also test 'CHICAGO Illinois' or something like that? Which would return 'University of Illinois Chicago' for example, but not return 'University of Chicago'.